content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -0,0 +1,195 @@
+```
+
+Subcommands includes:
+
+- `attach`:       Attach to a running container
+- `create`:       Create a new container
+- `exec`:         Run a command in a running container
+- `version`:      Display runtime version information
+- `images`:       List images
+- `inspect`:      Display the status of one or more containers
+- `inspecti`:     Return the status of one ore more images
+- `inspectp`:     Display the status of one or more pods
+- `logs`:         Fetch the logs of a container
+- `port-forward`: Forward local port to a pod
+- `ps`:           List containers
+- `pull`:         Pull an image from a registry
+- `runp`:         Run a new pod
+- `rm`:           Remove one or more containers
+- `rmi`:          Remove one or more images
+- `rmp`:          Remove one or more pods
+- `pods`:         List pods
+- `start`:        Start one or more created containers
+- `info`:         Display information of the container runtime
+- `stop`:         Stop one or more running containers
+- `stopp`:        Stop one or more running pods
+- `update`:       Update one or more running containers
+- `config`:       Get and set crictl options
+- `stats`:        List container(s) resource usage statistics
+- `completion`:   Output bash shell completion code
+- `help, h`:      Shows a list of commands or help for one command
+
+crictl connects to `unix:///var/run/dockershim.sock` by default. For other runtimes, the endpoint can be set in three ways:
+
+- By setting flags `--runtime-endpoint` and `--image-endpoint`
+- By setting environment variables `CONTAINER_RUNTIME_ENDPOINT` and `IMAGE_SERVICE_ENDPOINT`
+- By setting the endpoint in the config file `--config=/etc/crictl.yaml`
+
+```sh
+$ cat /etc/crictl.yaml
+runtime-endpoint: unix:///var/run/dockershim.sock
+image-endpoint: unix:///var/run/dockershim.sock
+timeout: 10
+debug: true
+```
+
+## Additional options
+
+- `--runtime-endpoint`, `-r`: CRI server runtime endpoint (default: "unix:///var/run/dockershim.sock").The default server is dockershim. If we want to debug other CRI server such as frakti, we can add flag `--runtime-endpoint=/var/run/frakti.sock`
+- `--image-endpoint`, `-i`: CRI server image endpoint, default same as runtime endpoint.
+- `--timeout`, `-t`: Timeout of connecting to server (default: 10s)
+- `--debug`, `-D`: Enable debug output
+- `--help`, `-h`: show help
+- `--version`, `-v`: print the version information of crictl
+- `--config`, `-c`: Config file in yaml format. Overrided by flags or environment variables.
+
+## Examples
+
+### Run pod sandbox with config file
+
+```sh
+$ cat pod-config.json
+{
+    "metadata": {
+        "name": "nginx-sandbox",
+        "namespace": "default",
+        "attempt": 1,
+        "uid": "hdishd83djaidwnduwk28bcsb"
+    },
+    "logDirectory": "/tmp",
+    "linux": {
+    }
+}
+
+$ crictl runp pod-config.json
+f84dd361f8dc51518ed291fbadd6db537b0496536c1d2d6c05ff943ce8c9a54f
+```
+
+List pod sandboxes and check the sandbox is in Ready state:
```

Comment:
This is the feature we want to demonstrate. Please include more sandboxes, and also include a fitler example.